client/src/components/AlertBanner/AlertBanner.stories.js

```diff
@@ -0,0 +1,24 @@
+import _ from "lodash";
```

Comment:
Unused lodash import? Check if your linter is flagging this, please fix any linter warnings/errors you have.

Worth noting, since story files aren't in the build, you won't see lint output for them from the build process. You'll have to keep an eye on the VS Code problem alerts:
![image](https://user-images.githubusercontent.com/11301919/118044013-1cf68c80-b344-11eb-9ab0-dabcd509ff5d.png)

Or run eslint from the console, specifically looking for `.stories.js` maybe.

Reply:
CI doesn't lint `client/src` explicitly if I remember right, only notices errors through the build output. If missing linter warnings in these non-build modules is common, we'll have to cover them in CI somehow.

Reply:
Fixed!